ruby/vampires.rb

```diff
@@ -0,0 +1,60 @@
+# Ask how many times to execute the loop
+puts "how many employees need to be processed?"
+to_process = gets.chomp
+
+counter = 1
+
+# iterates through counter until 'to_process' is reached.
+while counter <= to_process.to_i
+  # Ask new hire for name
+  puts "Employee #{counter}, what is your full name?"
+  name = gets.chomp
+
+  # Ask age of new hire
+  puts "How old are you? (numbers only please)"
+  age = (gets.chomp)
+
+  # Ask what year new hire was born
+  puts "What year were you born in?"
+  birth_year = gets.chomp
+
+  # Ask whether the new hire will eat garlic bread
+  puts "I hear they're serving garlic bread in the cafeteria today.  Sound good?"
+  garlic_response = gets.chomp
+
+  # Ask whether the new hire would like to enroll in company health insurance.
+  puts "Would you like to enroll in company insurance?"
+  insurance_choice = gets.chomp
+
+  # Ask employee about their allergies.
+  puts "Are you alergic to anything?  (when you're done listing allergies, please type \"done\")"
+  allergy = gets.chomp
+  while allergy != "done"
+    if allergy.downcase == "sunshine"
+      break
+    else
+      puts "any other allergies? (enter \"done\" when done)"
+      allergy = gets.chomp
+    end
+  end
+
+  if name == "Drake Cula" or name == "Tu Fang"

```

Comment:
Great work picking out the initial break case.  It is always important to focus on finding the most efficient document flow.

I think you could nest this further to come up with an even more efficient result.

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```